content/en/examples/access/admission-webhook-match-conditions.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+webhooks:
+- name: my-webhook.example.com
+  matchPolicy: Equivalent
+  rules:
+  - operations: ['CREATE','UPDATE']
+    apiGroups: ['*']
+    apiVersions: ['*']
+    resources: ['*']
+  failurePolicy: 'Fail'
```

```suggestion
  failurePolicy: 'Fail' # optional
```
(is that right?)

What's the default?